src/common/Header/Header.js

```diff
@@ -106,59 +105,59 @@ class Header extends Component {
-    }
+    };
     closeModalHandler = () => {
-        this.setState({modalIsOpen: false})
+        this.setState({modalIsOpen: false});
         // Login
-        this.setState({contactNoRequired: "dispNone"})
-        this.setState({passwordRequired: "dispNone"})
-        this.setState({value: 0})
+        this.setState({contactNoRequired: "dispNone"});
+        this.setState({passwordRequired: "dispNone"});
+        this.setState({value: 0});
         // Signup
-        this.setState({emailRequired: "dispNone"})
-        this.setState({firstnameRequired: "dispNone"})
+        this.setState({emailRequired: "dispNone"});
+        this.setState({firstnameRequired: "dispNone"});
         this.setState({signupErrorMsg: ""});
         this.setState({loginErrorMsg: ""})
 
-    }
+    };
     tabChangeHandler = (event, value) => {
         this.setState({value});
-        this.setState({contactNoRequired: "dispNone"})
-        this.setState({passwordRequired: "dispNone"})
-        this.setState({emailRequired: "dispNone"})
-        this.setState({firstnameRequired: "dispNone"})
-        this.setState({contactno: ""})
-        this.setState({password: ""})
-        this.setState({email: ""})
-        this.setState({firstname: ""})
+        this.setState({contactNoRequired: "dispNone"});
+        this.setState({passwordRequired: "dispNone"});
+        this.setState({emailRequired: "dispNone"});
+        this.setState({firstnameRequired: "dispNone"});
+        this.setState({contactno: ""});
+        this.setState({password: ""});
+        this.setState({email: ""});
+        this.setState({firstname: ""});
         this.setState({signupErrorMsg: ""});
         this.setState({loginErrorMsg: ""});
-    }
+    };
     loginClickHandler = () => {
-        this.state.password === "" ? this.setState({passwordRequired: "dispBlock"}) : this.setState({passwordRequired: "dispNone"})
+        this.state.password === "" ? this.setState({passwordRequired: "dispBlock"}) : this.setState({passwordRequired: "dispNone"});
         let isValidContactNo = this.contactnoFieldValidation();
         this.setState({
             loginErrorMsg: ""
-        })
+        });
         if (isValidContactNo === true && this.state.password !== "") {
             this.callApiForLogin()
         }
-    }
+    };
     callApiForLogin = () => {
-        console.log("Login api started")
+        console.log("Login api started");
         let xhrPosts = new XMLHttpRequest();
-        let that = this
+        let that = this;
 
         let param = window.btoa(this.state.contactno + ":" + this.state.password);
-        console.log(param)
+        console.log(param);
         xhrPosts.addEventListener("readystatechange", function () {
 
             if (this.readyState === 4) {
-                console.log(this.responseText)
-                console.log(this.status)
-                var data = JSON.parse(this.responseText)
+                console.log(this.responseText);
```

Comment:
Please remove the unnecessary console log statements

Reply:
Changes done.